app/views/application/_sidebar.html.erb

```diff
@@ -16,7 +16,7 @@
 
   <%= yield :sidebar %>
 
-  <% if current_user %>
+  <% unless current_user.try(:teams).blank? %>
```

Comment:
you are using `try(:teams)` again, maybe it is a better idea to create a method (if `teams` if not an array ;)